src/MoBi.Assets/AppConstants.cs

```diff
@@ -1878,6 +1878,20 @@ public static string AlsoImportIndividualsAndExpressions(int numberOfIndividuals
 
             return $"Also import {individual} and {expression}?";
          }
+
+         public static string CouldNotAddExpressionsDuplicatingMolecule(IReadOnlyList<string> allNames)
+         {
+            if(allNames.Count == 1)
+               return $"The expression could not be added because an expression is already selected for the protein{Environment.NewLine}{namesList(allNames)}";
+
+            return $"Expressions could not be added because expressions are already selected for the protein{Environment.NewLine}{namesList(allNames)}";
+         }
+
+         private static string namesList(IReadOnlyList<string> allNames)
+         {
+            var listBullet = $"{Environment.NewLine} - ";
```

Comment:
we typycally use a string builder and add newlines etc. (at least in PKSim). Not sure if this is better in terms of formatting